language/move-lang/src/expansion/ast.rs

```diff
@@ -360,6 +372,16 @@ impl SpecId {
     }
 }
 
+impl Neighbor {
+    pub fn mident(&self) -> ModuleIdent {
```

Comment:
Maybe something like `into_module_ident`

Or `module_ident` 

Either way I'd drop the clone

Reply:
Changed to `into_module_ident()` and dropped the clone.